drv/stm32h7-spi-server/build.rs

```diff
@@ -160,27 +182,31 @@ impl ToTokens for SpiConfig {
             .map(|(i, k)| (k, i))
             .collect();
 
+        // The svd2rust PAC can't decide whether acronyms are words, so we get
+        // to produce both identifiers.
+        let devname: syn::Ident =
+            syn::parse_str(&format!("SPI{}", self.controller)).unwrap();
+        let pname: syn::Ident =
+            syn::parse_str(&format!("Spi{}", self.controller)).unwrap();
+
         // We don't derive ToTokens for DeviceDescriptorConfig because it needs
         // extra knowledge (the mux_indices map) to do the conversion. Instead,
         // convert it here:
         let device_code = self.devices.values().map(|dev| {
             let mux_index = mux_indices[&dev.mux];
             let cs = &dev.cs;
+            let div: syn::Ident =
+                syn::parse_str(&format!("{:?}", dev.clock_divider)).unwrap();
             quote::quote! {
                 DeviceDescriptor {
                     mux_index: #mux_index,
                     cs: #cs,
+                    // `spi1` here is _not_ a typo/oversight.
```

Comment:
why?

Reply:
Because the PAC crate has decided to call all SPI types spi1. There are comments to this effect at several places in the driver, I can add another one if you like.

Reply:
I saw other mentions of spi1 being correct but not why.

Reply:
Would you like me to add a comment?

Reply:
Yes